chuckie.py

```diff
@@ -1,24 +1,24 @@
-import random
 import boto3
+from boto3.dynamodb.conditions import Key
+import os
 
 dynamodb = boto3.resource('dynamodb')
 table = dynamodb.Table('wotd')
 
 def lambda_handler(event, context):
-    word = table.get_item(
-        Key={
-            'language': 'spanish',
-            'id': random.randrange(1, 697)
-        }
+    wotd = table.query(
+        IndexName='language-word-index',
+        KeyConditionExpression=Key('language').eq('spanish') & Key('word').eq(os.environ['word'])
     )
-    wotd = '<speak>' + word["Item"]["word"] + ' which means ' + word["Item"]["word_translation"] + ' and sounds like <audio src="' + word["Item"]["word_sound"] + '"/></speak>'
+    item = wotd["Items"]
+    parsed = '<speak>The word of the day is <audio src="' + item[0]["word_sound"] + '"/> which means ' + item[0]["word_translation"] + '</speak>'
```

Comment:
Interesting choice of a name `parsed`.  To me, that is something we picked apart.  Here, we're assembling a final string.  Mabye `wordOfTheDayUtterance` or `utterance`, depending on what is more idiomatic Python.

Reply:
@kurron Utterances are different phrases people can say to activate Chuckie, not the response given by Alexa. https://developer.amazon.com/docs/custom-skills/custom-interaction-model-reference.html#sample-utterances-syntax I think that would end up being more confusing than ```parsed```.

Maybe something like ```output```, ```outputSpeech```, ```response```, or ```wotdResponse``` could be used, since they are more descriptive?

Reply:
I like `resonse`.  It fits the context and should be quickly grokked by the reader.  Excellent suggestion, @sarahmau .

Reply:
@kurron @sarahmau I already use `response` as a variable name, so I'm going to use `outputSpeech` as it is the most descriptive

Reply:
@fromcj ```outputSpeech``` it is! :)